packages/docs/celo-codebase/protocol/governance.md

```diff
@@ -13,23 +13,23 @@ The change procedure happens in the following phases:
 3.  Referendum
 4.  Execution
 
-{% hint style="info" %}
-**Note:** the timings mentioned in the rest of this section are for the Alfajores Testnet. It is expected that mainnet timings will be much longer to allow for proper proposal review and engagement.
-{% endhint %}
+### Overview of Phases
+
+Each proposal starts on the **Proposal Queue** where it may receive upvotes for promotion to the Approval phase. Proposals are promoted once per day, and must be promoted within 4 weeks. **Approval** lasts 1 days, during which the proposal must be approved by the Approver. Approved proposals are promoted to the Referendum stage. **Referendum** lasts five days, during which owners of Locked Celo vote yes or no on the proposal. Proposals that satisfy the necessary quorum are promoted to the execution phase. **Execution** lasts up to three days, during which anybody may trigger the execution of the proposal. 
 
 ### Proposal
 
-Any user may submit a “Proposal” to the `Governance` smart contract, along with a small deposit of CELO. This deposit is required to avoid spam proposals, and is refunded to the proposer if the community decides the proposal is worthy of voting on. A “Proposal” consists of a timestamp and the information needed to the “call” the operation code that will run if the proposal is accepted. This information includes an address, data, and value. Submitted proposals are added to the queue of proposals and expire from this list after one week.
+Any user may submit a Proposal to the `Governance` smart contract, along with a small deposit of CELO. This deposit is required to avoid spam proposals, and is refunded to the proposer if the proposal reaches the Approval stage. A Proposal consists of a list of transactions, and a description URL where voters can get more information about the proposal. Transaction data in the proposal includes the destination address, data, and value. If the proposal passes, the included transactions will be executed by the `Governance` contract.
```

Comment:
```suggestion
Any user may submit a Proposal to the `Governance` smart contract, along with a small deposit of CELO. This deposit is required to avoid spam proposals, and is refunded to the proposer if the proposal reaches the Approval stage. A Proposal consists of a list of transactions, and a description URL where voters can get more information about the proposal. It is encouraged that this description URL points to a CGP document in the [celo-org/celo-proposals](https://github.com/celo-org/celo-proposals) repository. Transaction data in the proposal includes the destination address, data, and value. If the proposal passes, the included transactions will be executed by the `Governance` contract.
```